docker_push

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#Push built docker image to dockerhub
+echo "$DOCKER_PASSWORD" | docker login -u kogoe1 --password-stdin
+docker push kogoe1/todo_app:latest
+
+# Get the latest image from Docker Hub
+docker pull kogoe1/todo_app:latest
+docker login --username=fiifiogoe@yahoo.com --password=$HEROKU_API_KEY registry.heroku.com
```

Comment:
You don't need to put a username in here, `_` will do and Heroku will work it out from the API key

Reply:
Thanks for the comments Ben. Your suggestion has been applied